lib/surrealist/hash_utils.rb

```diff
@@ -3,6 +3,8 @@
 module Surrealist
   # A helper class for hashes transformations.
   module HashUtils
+    EMPTY_HASH = {}.freeze
```

Comment:
`HashUtils` seems like a more obvious place to store this.

Reply:
Yep